RRemoteConfig/Logger.swift

```diff
@@ -17,10 +17,17 @@ internal class Logger {
         #endif
     }
 
+    /// Verbose
+    class func v(_ message: String) {
+        #if DEBUG
+        //print("🔍 \(message)")
```

Comment:
You need this print, right?

Reply:
No, that's for our own verbose logging which we want disabled by default. App teams will complain if it's enabled. I might make it switchable using a plist boolean in future.